apps/condo/domains/common/utils/mail.js

```diff
@@ -0,0 +1,59 @@
+const mailDomains = [
```

Comment:
It's better to use UPPER_CASE for global consts

Reply:
done